iOS/Views/MovieTableViewCell.swift

```diff
@@ -0,0 +1,42 @@
+import UIKit
+import Shared
+
+class MovieTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var movieBackdrop: UIImageView!
+    @IBOutlet weak var movieGenresImages: UIStackView!
+    @IBOutlet weak var movieReleaseDate: UILabel!
```

Comment:
Because it is a `@IBOutlet` and we are in the view class, no. But in other context could mean just Date. I do not like the hungarian notation, mainly because we are in a typed world :)